README.md

```diff
@@ -297,6 +297,14 @@ Config file locations:
 | Windows | %LOCALAPPDATA%/k8sgpt/k8sgpt.yaml                |
 
 
+Below is a sample config file content, in case the situation when no browser available:
+```
+backend_type: openai  # so far only openAI
+kubernetesclient: {}  # use default local kube-config when blank
+openai_key: $(yourKey)
```

Comment:
Hello 👋. The configuration format is wrong. We use providers struct.

Reply:
Yeah, I fully misunderstood the config. I tried to resolve the need of browser. I will update it